web/src/pages/Environments/EnvsModal.tsx

```diff
@@ -0,0 +1,39 @@
+import {Form, Modal} from 'antd';
+import {Dispatch, SetStateAction} from 'react';
+import styled from 'styled-components';
+import {IEnvironment} from '../../redux/apis/TraceTest.api';
+import {EnvironmentForm} from './EnvironmentForm';
+import {EnvironmentState} from './EnvironmentState';
+
+const CustomModal = styled(Modal)`
+  && {
+    .ant-modal-footer {
+      display: none;
+    }
+  }
+`;
+
+interface IProps {
+  state: EnvironmentState;
+  setState: Dispatch<SetStateAction<EnvironmentState>>;
+}
+
+export const EnvsModal: React.FC<IProps> = ({state, setState}) => {
```

Comment:
What's the difference between this and the one at the prev file?

Reply:
duplicated file while renaming ✅